hudi-common/src/test/java/org/apache/hudi/common/model/TestOverwriteWithLatestAvroPayload.java

```diff
@@ -80,13 +85,16 @@ public void testDeletedRecord() throws IOException {
     record1.put("id", "1");
     record1.put("partition", "partition0");
     record1.put("ts", 0L);
-    record1.put("_hoodie_is_deleted", false);
+    record1.put(defaultDeleteField, false);
```

Comment:
can we also fix lines 70 and 71. While creating the OverwriteWithLatestAvroPayload, pass in the delete field marker. Again, can we test for both default and user defined. So, since these are active field, existing assertions should work.